src/accesspattern/datasubset.jl

```diff
@@ -1,7 +1,12 @@
 target(data) = target(identity, data)
 target(f, data) = f(data)
 
+
+
 target{N}(f, tuple::NTuple{N}) = target(f, tuple[N])
+target{N}(f, tuple::NTuple{N}, obsdim::NTuple{N}) = target(f, tuple[N], obsdim[N])
```

Comment:
Well, that may not work actually. Maybe let's add another method signature, where `obsdim::LearnBase.ObsDimension`